pkgs/development/python-modules/pwntools/python3.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, Mako, packaging, pysocks, pygments, ROPGadget
+, capstone, paramiko, pip, psutil
+, pyelftools, pyserial, dateutil
+, requests, sortedcontainers, tox, unicorn
+, intervaltree }:
+
+
+buildPythonPackage rec {
+  version = "dev3";
+  pname = "pwntools";
+
+  src = fetchFromGitHub {
+    owner = "Gallopsled";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "130qar90i0vl3s1yqahf3m31mlsmbgj1lnqhdmncwyzhjzbp0x33";
+  };
+
+  propagatedBuildInputs = [
+		Mako packaging pysocks pygments ROPGadget capstone paramiko pip psutil pyelftools pyserial dateutil requests sortedcontainers tox unicorn intervaltree
```

Comment:
.... wonder why the project has pip listed in install_requires... odd

Reply:
https://github.com/Gallopsled/pwntools/commit/9ec1ea085260707051caab89762c1ba528689f79

Feels like it confuses environment requirements and packages dependencies... 🤷‍♂️

Reply:
what is that PR.... 250+ commits, 50K+ lines...

Reply:
Pwntools assumes a normal Ubuntu/Debian-like setup, so afaik the dependencies aren't really looked at much except when they don't work.

Some of those packages are only in nixpkgs because they are used as dependencies for pwntools, so it might be worthwhile going through and updating them as well.

Reply:
Although I understand your point, I don't think it's worth spending time going through these dependencies to update their `nixpkgs` counterpart.

Doing that upfront is tedious, and IMHO don't bring a lot of value.

Indeed, if a dependency needs to be updated for `pwntools` to work, it should be enforced in their `setup.py`, telling us which dependency to update for a new version to build in nixpkgs.